code/modules/atmospherics/machinery/atmosmachinery.dm

```diff
@@ -67,6 +67,8 @@
 
 /obj/machinery/atmospherics/examine(mob/user)
 	. = ..()
+	. += "<span class='notice'>[src] is on layer [piping_layer].</span>"
+	. += "<span class='notice'>[src] is of [GLOB.pipe_color_name[pipe_color]] color"
```

Comment:
```suggestion
	. += "<span class='notice'>[src] is [GLOB.pipe_color_name[pipe_color]]."
```

This reads better with all the possible colors.

Reply:
coolio